### PR TITLE
TA-487 Tags on tasks are confusing and useless

### PR DESCRIPTION
**Pull request by nttuyen**

### CODE REVIEW

File: services/src/main/resources/db/changelog/task.db.changelog-1.0.0.xml [hunk 1]
```diff
@@ -403,4 +403,19 @@
...
+      <createTable tableName="TASK_TAGS">
+        <column name="TASK_ID" type="BIGINT">
+          <constraints foreignKeyName="FK_TASK_TAGS_TASK_TASKS_01" references="TASK_TASKS(TASK_ID)" nullable="false"/>
+        </column>
+        <column name="TAG" type="NVARCHAR(50)">
+          <constraints nullable="false"/>
+        </column>
+      </createTable>
+    </rollback>
+  </changeSet>
```
Comment (trongtt) [inline services/src/main/resources/db/changelog/task.db.changelog-1.0.0.xml]: I think we should move this change to a specific file based on the version number we start having this change ?

File: task-management/src/main/java/org/exoplatform/task/management/assets/javascripts/editinline.js [hunk 2]
```diff
@@ -700,30 +700,7 @@ define('ta_edit_inline',
...
-                                },
-                                success: function(res) {
-                                    callback(res);
-                                }
-                            });
-                        }
-                    };
-
-                } else if (fieldName == 'labels') {
+                if (fieldName == 'labels') {
```
Comment (trongtt) [inline task-management/src/main/java/org/exoplatform/task/management/assets/javascripts/editinline.js]: can we merge this code block for "labels" with the one we have above ?

File: services/src/main/resources/db/changelog/task.db.changelog-1.0.0.xml (hunk 1 above)
Comment (thomasdelhomenie) [inline services/src/main/resources/db/changelog/task.db.changelog-1.0.0.xml]: You mean in a file task.db.changelog-1.1.0.xml ? Sounds like a good idea.

**Comment by thomasdelhomenie** [comment]
I had a look and did a test. No additional remarks than the one from @trongtt .

File: services/src/main/resources/db/changelog/task.db.changelog-1.0.0.xml (hunk 1 above)
Comment (trongtt) [inline services/src/main/resources/db/changelog/task.db.changelog-1.0.0.xml]: Yes, if we intend to have this in next 1.1.0 release version.

Comment (thomasdelhomenie) [inline services/src/main/resources/db/changelog/task.db.changelog-1.0.0.xml]: 1.1.0 will probably be the next version yes, so let's go for task.db.changelog-1.1.0.xml.

Comment (nttuyen) [inline services/src/main/resources/db/changelog/task.db.changelog-1.0.0.xml]: I added task.db.changelog-1.1.0.xml and move the change-set to remove table TAG to this file.

File: task-management/src/main/java/org/exoplatform/task/management/assets/javascripts/editinline.js (hunk 2 above)
Comment (nttuyen) [inline task-management/src/main/java/org/exoplatform/task/management/assets/javascripts/editinline.js]: They are merged now

**Comment by nttuyen** [comment]
Merged!

**Pull request closed without merging**